KthSmallestInSortedMatrix.java

```diff
@@ -0,0 +1,27 @@
+/*
+method- 1
+use max priority queue(max heap) to find the kth smallest element
+time complexity : O(m x n), worst case end up traversing entrire matrix
```

Comment:
you should be considering the sorting that takes place in priority queue as well. It should be mnlogmn